_episodes/03-tools.md

```diff
@@ -10,57 +10,71 @@ objectives:
   - Choose the right tool for the right reason.
 keypoints:
   - Some popular solutions make reproducibility and maintenance of multiple code versions difficult.
-  - Do not reinvent the wheel - learn from large and busy open source projects.
 ---
 
-## Comments in the source code
+## What tools and solutions are out there?
 
-- Good for programmers
-- Typically not sufficient for users
-- Can be used to auto-generate documentation for functions/classes
-- Version controlled alongside code
+- [Comments in the source code](#comments-in-the-source-code)
+- [README files in the source tree](#readme-files-in-the-source-tree)
+- [Wikis](#wikis)
+- [LaTeX/PDF](#latexpdf)
+- [Doxygen](#doxygen)
+- [reStructuredText and Markdown](#restructuredtext-and-markdown)
+- [HTML static site generators](#html-static-site-generators)
 
 ---
 
-## README files in the source tree
+### Comments in the source code
+
+- Advantages
+  - Good for programmers
+  - Version controlled alongside code
+  - Can be used to auto-generate documentation for functions/classes
+- Disadvantage
+  - Probably not enough for users
+
+---
+
+### README files in the source tree
 
 - Advantage
   - Versioned (goes with the code development)
+  - It is often good enough to have a `README.md` or `README.rst` along with your code/script
 - Disadvantage
   - You need a terminal or GitHub/GitLab browser to read them
+  - Sometimes users have no access to the source tree
 - If you use README files, use either
   [RST](http://docutils.sourceforge.net/rst.html) or
   [Markdown](http://daringfireball.net/projects/markdown/) markup
```

Comment:
```suggestion
  [Markdown](https://commonmark.org/help/) markup
```
This is a personal crusade, so feel free to ignore.  I claim markdown is a dead language, since the original author doesn't want any actual standard called markdown - so main species is dead, and we have many different strains which can never converge. (so, it's Markdown that is dead, not markdown-the-concept). CommonMark is the name of the people who want some sort of standard, and this link seems non-advertized and non-commercial, the closest we can get to an attempt at a standard, and not bad.  The interactive tutorial is really good, too.
It doesn't have lots of advanced features, though.  But I guess those vary so much...

Reply:
Thanks for clarification for me!